test/any/files-helper.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+// Manually require code under test (in alphabetical order).
+//
+// TODO(markdittmer): Move to FOAM classloader for this.
+
+beforeAll(function() {
+  // Use of foam.isServer = <running in Node JS?>:
+  // - Do not load Angular-related in Node JS (foam.isServer = true).
+  // - Do not load import-from-filesystem in browser.
+
+  if (!foam.isServer) {
```

Comment:
Maybe having two subhelpers for importing server-only and client-only stuff would be a better structure?